brave/brave-core
# Adds personaId, userId and bootStamp to rewards internal page

Resolves https://github.com/brave/brave-browser/issues/3624

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on

## Test Plan:
- enable rewards
- go to brave://rewards-internals/
- make sure that you see Persona ID, User ID and Wallet created field
- make sure that this fields match ledger_state values

## Reviewer Checklist:
